source/MaterialXGenShader/Util.cpp

```diff
@@ -320,6 +320,11 @@ bool requiresImplementation(ConstNodeDefPtr nodeDef)
     {
         return false;
     }
+    static string ORGANIZATION_STRING("organization");
+    if (nodeDef->getNodeGroup() == ORGANIZATION_STRING)
+    {
+        return false;
+    }
     static string TYPE_NONE("none");
     const string& typeAttribute = nodeDef->getType();
     return !typeAttribute.empty() && typeAttribute != TYPE_NONE;
```

Comment:
Do you really need the new test for "organization" above?

Is this test for type not enough, either having an empty type or type=="none"?

Reply:
The new node seems to have a type. Let me take a look. Perhaps I need to specifically set the type to "none" ?

Reply:
Ah yes, if no type is set I think MaterialXCore will default to "color3" or something like that.